aboutme.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="Description" content="INF 7420 Project">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kim Poma - Portfolio</title>
+</head>
+<body>
+    <header>
+       <nav>
+           <img src="images/logo.png"alt="Logo of tree with child reading underneath it">
```

Comment:
Hi @kimp7777 there needs to be a space between attributes. Instead of `<img src="images/logo.png"alt="Logo of tree with child reading underneath it">` it should be `<img src="images/logo.png" alt="Logo of tree with child reading underneath it">` (a space before the alt attribute). This issue is repeated, please use the View > Problems option in VS Code to point you to each instance of this issue.